homeassistant/components/otbr/config_flow.py

```diff
@@ -43,6 +39,12 @@ class AlreadyConfigured(HomeAssistantError):
     """Raised when the router is already configured."""
 
 
+@callback
+def get_addon_manager(hass: HomeAssistant, slug: str) -> AddonManager:
```

Comment:
Each add-on manager should be a singleton per add-on. We can use the singleton decorator. See other integrations that create add-on managers.